Workbooks/Azure Resources/Azure Resource Locations/settings.json

```diff
@@ -0,0 +1,8 @@
+{
+    "$schema": "https://github.com/Microsoft/Application-Insights-Workbooks/blob/master/schema/settings.json",
+    "name":"Azure Resource Locations",
+    "author": "Microsoft",
+    "galleries": [
+        { "type": "workbook", "resourceType": "Azure Monitor", "categoryKey": "All", "order": 100  }
```

Comment:
You already have a categoryResources.json at the lower level - you don't need a categoryKey. The category string (Azure Resources) will be picked from categoryResources.json.

Reply:
categorykey from settings.json